### ISSUE TITLE
When running Example getting TypeError: 'staticmethod' object is not callable

### ISSUE THREAD

**Issue by arpans2112** [3b1b/manim#1945]
### Describe the error
i have install the manim as per the instruction given on the documentation, when running the example getting TypeError: 'staticmethod' object is not callable 

### Code and Error
**Code**:
manimgl example_scenes.py OpeningManimExample     

Traceback (most recent call last):
  File "/Users/arpansaini/opt/anaconda3/bin/manimgl", line 5, in <module>
    from manimlib.__main__ import main
  File "/Users/arpansaini/Documents/Techsqually/projects/maniumProj/manim/manimlib/__init__.py", line 14, in <module>
    from manimlib.animation.animation import *
  File "/Users/arpansaini/Documents/Techsqually/projects/maniumProj/manim/manimlib/animation/animation.py", line 5, in <module>
    from manimlib.mobject.mobject import _AnimationBuilder
  File "/Users/arpansaini/Documents/Techsqually/projects/maniumProj/manim/manimlib/mobject/mobject.py", line 58, in <module>
    class Mobject(object):
  File "/Users/arpansaini/Documents/Techsqually/projects/maniumProj/manim/manimlib/mobject/mobject.py", line 555, in Mobject
    def serialize(self):
TypeError: 'staticmethod' object is not callable

**Error**:

### Environment
**MAC OS**: 
**manim version**: master 
**python version**:3.9.12

**Comment by Pi-31415** [comment]
Same here too.

**Comment by Pi-31415** [comment]
It works after I removed the static decorators in manim/manimlib/mobject/mobject.py and manim/manimlib/mobject/types/vectorized_mobject.py.

I submitted a pull request. Let's see.

**Comment by TonyCrane** [comment]
Fixed in #1951

**Comment by yaoyhu** [comment]
For newbies like me who encountered this problem:  you may have installed the package by cloning the library. Check your env library by command `pip list` to see if your `manimgl/manimlib` is suffixed by your own path like `home/name/manim`.

If it is, uninstall them and reinstall directly via pip.

```shell
pip uninstall manimgl manimlib
pip install manimgl manimlib
```

> Quote from README.md: If you want to hack on manimlib itself, clone this repository and in that directory execute:
